test/testcafe/spec/SelectAuthenticatorForVerification_spec.js

```diff
@@ -251,6 +258,137 @@ test.requestHooks(mockChallengePassword)('should load select authenticator list'
   await t.expect(await selectFactorPage.signoutLinkExists()).eql(true);
 });
 
+test.requestHooks(mockChallengeWithNickname)('should load select authenticator list with nicknames if available', async t => {
```

Comment:
```suggestion
test.meta('v3', false).requestHooks(mockChallengeWithNickname)('should load select authenticator list with nicknames if available', async t => {
```

Reply:
Want to disable this test for v3 since there is a ticket to add this logic into V3.